<launch>

    <group ns="robot1/cloud_map_merge">
        <param name="init_pose_x" value="0.0"/>
        <param name="init_pose_y" value="0.0"/>
        <param name="init_pose_yaw" value="0.0"/>
    </group>

    <group ns="robot2/cloud_map_merge">
        <param name="init_pose_x" value="0.0"/>
        <param name="init_pose_y" value="12.0"/>
        <param name="init_pose_yaw" value="0.0"/>
    </group>

    <node pkg="cloud_map_merge" type="cloud_map_merge_test" name="cloud_map_merge_test">
        <param name="map_topic" value="rtabmap/cloud_obstacles"/>
        <param name="merge_map_frame" value="merge_map"/>
        <param name="param_namespace" value="cloud_map_merge"/>
        <param name="ceiling_height" value="2.4"/>
        <param name="floor_height" value="-0.05"/>
    </node>

</launch>